README.md

```diff
@@ -23,6 +23,15 @@ Follow these steps to start a local Django server using a PostgreSQL database:
 [Windows/Mac](http://www.postgresqltutorial.com/install-postgresql/)
```

Comment:
Remove the Mac from this link text:

```suggestion
[Windows](http://www.postgresqltutorial.com/install-postgresql/)
```

Then below that line and above Linux, add this link:
```markdown
  [Mac](https://github.com/aggie-coding-club/Automatic-Aggie-Scheduler/wiki/Setup-Postgres-PGAdmin-on-MacOs)
```

Which creates a hyperlink that looks like [Mac](https://github.com/aggie-coding-club/Automatic-Aggie-Scheduler/wiki/Setup-Postgres-PGAdmin-on-MacOs)